tensorflow/compiler/mlir/lite/tests/end2end/BUILD

```diff
@@ -8,6 +8,12 @@ glob_lit_tests(
         ":test_utilities",
     ],
     driver = "@llvm-project//mlir:run_lit.sh",
+    tags_override = {
```

Comment:
Do you expect rocm to be used tflite side? if not, making it a default tag for all the tests might be better.

Reply:
>Do you expect rocm to be used tflite side? 

don't know the answer to that question.  /cc @whchung 

w.r.t. making `no_rocm` tag the default, I had rather not do that, while we wait for clarity on ROCm usage in tflite/mlir. hope that is okay.

Reply:
That's reasonable yes.